core/services/gateway/connector/connector.go

```diff
@@ -179,7 +180,12 @@ func (c *gatewayConnector) AwaitConnection(ctx context.Context, gatewayID string
 }
 
 func (c *gatewayConnector) SendToGateway(ctx context.Context, gatewayID string, msg *api.Message) error {
```

Comment:
signature of this function has been changed in `develop`. I would recommend merging `develop` because a lot of gateway code has been changed :)